README.md

```diff
@@ -10,30 +9,49 @@ README
 What is RollerworksSearch?
 --------------------------
 
-RollerworksSearch provides you with a light yet powerful search system
-to easily integrate into your PHP application.
+RollerworksSearch provides you with a powerful search system
+for your PHP application.
 
 The system has a modular design and can work with any PHP framework,
 user locale, data format or storage system.
 
+Whether you want to simply search for users in your SQL database or want to
+provide a powerful search system for searching products using an ElasticSearch
+back-end, everything is possible.
+
+Say your customer data is stored in the "customer" table while the "invoice"
+data is stored in it's own table. And invoice rows also have there own table.
+
+Searching (using the FilterQuery syntax) can be as simple as:
+`invoice_price: >"$20.00" invoice_row: ~*"my cool product"; customer_type: !consumer.`
```

Comment:
this example makes me understand what could this be used for, i think more of these examples could be good.

do you have custom hard coded queries like that? or is this more for a dashboard admin to be able to create their own user facing query builder so they can sift data through it? I have seen this done in jira for example to sift tickets. Is this the idea? if so you should even put this example and what other usages it maybe have.